app/models/__init__.py

```diff
@@ -6,3 +6,4 @@
 from agency import *  # noqa
 from user import *  # noqa
 from incident_report import *  # noqa
+from miscellaneous import *  # noqa
```

Comment:
@nancywong Not sure but seemed a bit odd to give this one model it's own python file. What do you think?